src/containers/MasterWindow.js

```diff
@@ -114,13 +111,13 @@ class MasterWindowContainer extends Component {
 
   getTabRow(tabId, rowId) {
     const { params } = this.props;
-    return getData(
-      'window',
-      params.windowType,
-      params.docId,
-      tabId,
-      rowId
-    ).catch(() => ({ rowId, tabId }));
+    return getData({
+      entity: 'window',
+      docType: params.windowType,
+      docId: params.docId,
+      tabId: tabId,
```

Comment:
can;t this be replaced with short version ? just tabId,      -- Same for the other one rowId,

Reply:
Theoretically yes, it shall be a constant.
Practically the `getData` swiss knife method is poor design.
IMHO we shall have a dedicated method getting 'window' data, 'view' data, 'board' data etc.